sites/main-site/src/pages/join.astro

```diff
@@ -145,6 +145,10 @@ const headings = [
             ><i class="fab fa-slack"></i> Get an invite to nf-core Slack</a
         >
     </p>
+    <div class="alert alert-info text-center">
+        If the invite link doesn't work because it has expired, please email us at
+        <a href="mailto:core@nf-co.re">core@nf-co.re</a> so that we can update it. Thanks!
+    </div>
```

Comment:
```suggestion
    <div class="alert alert-secondary text-center">
        If the invite link doesn't work, please email us at
        <a href="mailto:core@nf-co.re">core@nf-co.re</a> so that we can update it. Thanks!
    </div>
```